napari/_qt/qt_dims.py

```diff
@@ -280,106 +245,33 @@ def _trim_sliders(self, number_of_sliders):
-        axis_label = self.axis_labels.pop(index)
-        self.layout().removeWidget(axis_label)
-        slider.deleteLater()
-        axis_label.deleteLater()
+        self.layout().removeWidget(slider_widget)
+        slider_widget.deleteLater()
         nsliders = np.sum(self._displayed_sliders)
         self.setMinimumHeight(nsliders * self.SLIDERHEIGHT)
         self.last_used = None
 
-    def _create_range_slider_widget(self, axis: int):
-        """Creates a range slider widget for a given axis.
-
-        Parameters
-        ----------
-        axis : int
-            axis index
-
-        Returns
-        -------
-            slider : range slider
-        """
-        _range = self.dims.range[axis]
-        # Set the maximum values of the range slider to be one step less than
-        # the range of the layer as otherwise the slider can move beyond the
-        # shape of the layer as the endpoint is included
-        _range = (_range[0], _range[1] - _range[2], _range[2])
-        point = self.dims.point[axis]
-
-        slider = ModifiedScrollBar(Qt.Horizontal)
-        slider.setFocusPolicy(Qt.NoFocus)
-        slider.setMinimum(_range[0])
-        slider.setMaximum(_range[1])
-        slider.setSingleStep(_range[2])
-        slider.setPageStep(_range[2])
-        slider.setValue(point)
-
-        # Listener to be used for sending events back to model:
-        def slider_change_listener(value):
-            self.dims.set_point(axis, value)
-
-        # linking the listener to the slider:
-        slider.valueChanged.connect(slider_change_listener)
-
-        def slider_focused_listener():
-            self.last_used = self.sliders.index(slider)
-
-        # linking focus listener to the last used:
-        slider.sliderPressed.connect(slider_focused_listener)
-
-        return slider
-
-    def _create_axis_label_widget(self, axis):
-        """Create the axis label widget which accompanies its slider.
-
-        Parameters
-        ----------
-        axis : int
-            axis index
-
-        Returns
-        -------
-        label : QLabel
-            A label with the given text
-        """
-        label = QLineEdit()
-        label.setText(self.dims.axis_labels[axis])
-        label.home(False)
-        label.setToolTip('Axis label')
-        label.setAcceptDrops(False)
-        label.setEnabled(True)
-
-        def changeText():
-            with self.dims.events.axis_labels.blocker():
-                self.dims.set_axis_label(axis, label.text())
-            label.clearFocus()
-            self.setFocus()
-
-        label.editingFinished.connect(changeText)
-        return label
-
     def focus_up(self):
         """Shift focused dimension slider to be the next slider above."""
         displayed = list(np.nonzero(self._displayed_sliders)[0])
         if len(displayed) == 0:
             return
 
         if self.last_used is None:
+            # this code may be unreachable?
```

Comment:
`last_used` is initialized as `None`, but maybe better initialize it as `0` or something? I'm not sure if it is ever set to `None` again or if it is overwritten right away. If we change the initialization then fine to drop these

Reply:
I had noticed those lines when trying to get 100% test coverage: there didn't seem to be any way I could get to them. I think it must be overwritten right away.  I'll look at the initialization and trim if possible.

Reply:
I'm just gonna remove those comments.  I can't be 100% sure we have all tests cases covered and those conditionals aren't not hurting anything.